http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -367,6 +367,22 @@ public void testRequestObjectConfigMissingENCValue() throws Exception {
         testRequestObjectInvalidConfiguration(getOidcConfigurationInputStreamWithoutEncValue(REQUEST.getValue(), RSA_OAEP), RequestObjectErrorType.MISSING_ENC_VALUE);
     }
 
+    @Test
+    // Generate an invalid sessionRandomValue so that the nonce check fails
+    public void testSessionIdNonceMismatch() throws Exception {
+        performAuthentication(
+                getOidcConfigurationInputStreamWithRequestParameter(REQUEST.getValue(), NONE, "", ""),
+                KeycloakConfiguration.ALICE, KeycloakConfiguration.ALICE_PASSWORD,true,
+                HttpStatus.SC_FORBIDDEN,null, CLIENT_PAGE_TEXT, true);
+    }
+
+    @Test
+    public void testtestOpenIDScopeNONCE() throws Exception {
+        String expectedScope = OIDC_SCOPE;
```

Comment:
We don't need to test the expected scope here. The main idea is to just use configuration that doesn't make use of the request parameter so we can test what happens when a normal authentication request is A standard config like getOidcConfigurationInputStreamWithProviderUrl could be used.

Reply:
done